src/test/java/jenkins/plugins/github/api/SmokeTest.java

```diff
@@ -19,6 +21,10 @@
 import static org.junit.Assert.assertThat;
 
 public class SmokeTest {
+
+    @Rule
+    public JenkinsRule jenkins = new JenkinsRule();
```

Comment:
Why?

Reply:
Because tests were failing without it?

Reply:
Existing tests checking only library itself, where Jenkins instance is neeed?

Reply:
Yes I get that, but my question is _why_ tests would have been failing. Does this upgrade somehow introduce calls to `Jenkins.get` into tests which otherwise do not mention it? Or were they failing before this patch?

So now I see errors such as

```
java.lang.IllegalStateException: Jenkins.instance is missing. Read the documentation of Jenkins.getInstanceOrNull to see what you are doing wrong.
	at jenkins.model.Jenkins.get(Jenkins.java:758)
	at jenkins.telemetry.impl.StaplerDispatches$StaplerTrace.record(StaplerDispatches.java:105)
	at org.kohsuke.stapler.EvaluationTrace$ApplicationTracer.trace(EvaluationTrace.java:77)
	at org.kohsuke.stapler.Dispatcher.anonymizedTraceEval(Dispatcher.java:76)
	at org.kohsuke.stapler.Stapler.tryInvoke(Stapler.java:685)
	at org.kohsuke.stapler.Stapler.invoke(Stapler.java:870)
	at org.kohsuke.stapler.Stapler.invoke(Stapler.java:668)
	at org.kohsuke.stapler.Stapler.service(Stapler.java:238)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at …
```

which means that https://github.com/jenkinsci/github-api-plugin/blob/356484ad3f26f20d6a8f0c47a2196db188bb0e89/src/test/java/jenkins/plugins/github/api/mock/MockGitHub.java#L61 plus https://github.com/jenkinsci/jenkins/pull/3688 is causing problems and suggests that you need to update to 2.150.x to pick up https://github.com/jenkinsci/jenkins/pull/3712.

Reply:
Awesome!  Thanks @jglick !